pkg/whitesource/scanNPM.go

```diff
@@ -66,6 +66,7 @@ func (s *Scan) writeWhitesourceConfigJSON(config *ScanOptions, utils Utils, devD
 		setValueAndLogChange(npmConfig, "projectName", config.ProjectName)
 	}
 	setValueAndLogChange(npmConfig, "devDep", devDep)
+	setValueAndLogChange(npmConfig, "includeDevDependencies", config.NpmIncludeDevDependencies)
```

Comment:
Is this related to logging or should that maybe go into a dedicated PR?

Reply:
I can rename this PR to reflect the change additionally. This is a new parameter - adding to the step